source/server/http/admin.cc

```diff
@@ -653,7 +663,12 @@ void AdminFilter::onComplete() {
   Buffer::OwnedImpl response;
   Http::HeaderMapPtr header_map{new Http::HeaderMapImpl};
   RELEASE_ASSERT(request_headers_);
-  Http::Code code = parent_.runCallback(path, *request_headers_, *header_map, response);
+  Http::Code code = parent_.runCallback(path, *request_headers_, *header_map, response, *this);
+  bool end_stream = true;
+
+  if (path.find("/hystrix_event_stream") != std::string::npos) {
```

Comment:
You may already be planning to do this, but can we get rid of the special case by allowing the handler to be able to set the `end_stream` variable (we can still default it to `true`)?